### ISSUE TITLE
Signal trap not working

### ISSUE THREAD

**Issue by the729** [edgelesssys/ego#190]
```
func main() {
	c := make(chan os.Signal, 2)
	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
	s := <-c

	fmt.Printf("Traped signal: %v", s)
}
```
Signal trap not working. Without it, we can not implement graceful shutdown. Is there any recommendations?

**Comment by thomasten** [comment]
Yes, signals are not supported. I'll add that to the known limitations docs.

Can you describe your usage scenario a bit more? Maybe we can come up with an alternative solution.

**Comment by the729** [comment]
The use case is graceful shutdown. We want to trap syscall.SIGINT and syscall.SIGTERM, so the program can shutdown gracefully either with Ctrl+C or docker stop.

**Comment by thomasten** [comment]
Maybe you could adapt this workflow such that you send an exit command via a socket connection before stopping the program.
